microsoft_teams/icon_microsoft_teams/triggers/new_message_received/trigger.py

```diff
@@ -196,7 +196,8 @@ def get_indicators(self, message: str) -> object:
                 "ipv6_addresses": self.remove_duplicates(self.extract_ipv6_addresses(message))
             },
             "mac_addresses": self.remove_duplicates(self.extract_macs(message)),
-            "cves": self.remove_duplicates(self.extract_cve(message))
+            "cves": self.remove_duplicates(self.extract_cve(message)),
+            "uuid": self.remove_duplicates(self.extract_uuid(message))
```

Comment:
Should be `uuids` to be plural like the rest of the indicators

```suggestion
            "uuids": self.remove_duplicates(self.extract_uuid(message))
```

We also need to add `uuids` to the indicators object in the plugin.spec.yaml and regenerate - looks like the schema wasn't updated